apps/provider-console/src/utils/dateUtils.ts

```diff
@@ -54,3 +53,64 @@ export function getPrettyTime(timeMs: number): string {
     return `${Math.floor(timeMs / 1_000 / 60 / 60)}h ${roundDecimal(timeMs / 1_000 / 60, 2) % 60}m`;
   }
 }
+
+export function getPrettyTimeFromSeconds(seconds: number) {
```

Comment:
It's basically takes seconds and convert to pretty time like if you give 300 seconds then it returns 5 minutes,
if it's more than that then give like 2 hous and 30 minutes.
This is used to determine time left in deployment and show them meaningful data instead of blocks.

Nothing done here, let me know if you want to change something.